在Android17之前的项目如何使用Amigo

当制作使用android 17(4.2)之前的版本的APK的时候会报错 AndroidManifest.xml:29: error: No resource identifier found for attribute 'supportsRtl' in package 'android'，怎么在4.0集成Amigo？

另外在gradle3.3出现这个错误
16:32:41.366 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Caused by: java.lang.NullPointerException: Cannot invoke method entries() on null object
16:32:41.366 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at me.ele.amigo.AmigoPlugin.collectMultiDexInfo(AmigoPlugin.groovy:150)
16:32:41.366 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at me.ele.amigo.AmigoPlugin$_apply_closure1$_closure5$_closure7$_closure10.doCall(AmigoPlugin.groovy:114)
16:32:41.366 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:596)
16:32:41.366 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:577)
16:32:41.366 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:95)
16:32:41.366 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:76)
16:32:41.367 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	... 78 more

--- a/amigo-lib/src/main/AndroidManifest.xml
+++ b/amigo-lib/src/main/AndroidManifest.xml
@@ -1,9 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="me.ele.amigo">
 
-    <application
-        android:allowBackup="true"
-        android:supportsRtl="true">
+    <application>
 
         <service
             android:name=".AmigoService"

